src/index_runner/main.py

```diff
@@ -30,8 +30,20 @@
 def _handle_msg(msg):
     event_type = msg.get('evtype')
     if not event_type:
-        logger.warning(f"Missing 'evtype' in event: {msg}")
-        return
+        msg = f"Missing 'evtype' in event: {msg}"
+        logger.error(msg)
+        raise RuntimeError(msg)
+    objtype = msg.get('objtype')
+    if objtype is not None and isinstance(objtype, str) and len(objtype) > 0:
```

Comment:
objtype won't always be set, for example in an event for a workspace deletion. In these cases, we just continue.